src/index.test.tsx

```diff
@@ -0,0 +1,5 @@
+describe("P5Wrapper", () => {
+  it("tests the component", () => {
+    // test case should be here
```

Comment:
Can you add a test here?

Reply:
Yes, I can. Maybe, later in the evening. I need to figure out how to properly mock the canvas.

Reply:
We could have a sketch fixture for use across tests and then just render and assert on the tender with testing library for example?

Reply:
Sounds like a good idea. Thanks!